### PR TITLE
Chore: reformat large sections of code

### PR DESCRIPTION
**Pull request by dbmalkovsky**
This file had various and different formatting styles.  It appears that some of which was caused by removing blocks of code with comments (now removed).

The only significant code changes were adding the ".msg" suffix to two property bundle keys to match the most recent push of language_en.properties.

### CODE REVIEW

File: src/main/java/org/isf/admission/gui/AdmissionBrowser.java [hunk 1]
```diff
@@ -1741,422 +1700,376 @@ private JButton getSaveButton() {
...
+	    Date beginDate;
+	    Date endDate;
+	    if (admission.getAdmDate() != null)
+		    beginDate = admission.getAdmDate().getTime();
+	    else
+		    beginDate = null;
+	    if (admission.getDisDate() != null)
+		    endDate = admission.getDisDate().getTime();
+	    else
+		    endDate = null;
```
Comment (mwithi) [inline src/main/java/org/isf/admission/gui/AdmissionBrowser.java:2045]: code block missing?

Comment (dbmalkovsky) [reply src/main/java/org/isf/admission/gui/AdmissionBrowser.java:2045]: Yeah; the reformatting doesn't automatically add them; I will do that the next time through the file.

Comment (dbmalkovsky) [reply src/main/java/org/isf/admission/gui/AdmissionBrowser.java]: It was easy to do now so I did it.

Comment (mwithi) [reply src/main/java/org/isf/admission/gui/AdmissionBrowser.java]: Alright, thanks!

**Pull request merged**